src/components/PostList/PostList.jsx

```diff
@@ -1 +1,9 @@
-export const PostList = () => <>Put the list here</>;
+import { PostInfo } from '../PostInfo/PostInfo';
+
+export const PostList = ({ posts }) => {
+  return (
+    <div className="PostList">
+      {posts.map(post => post && <PostInfo post={post} key={post.id} />)}
```

Comment:
post cannot be undefined
```suggestion
      {posts.map(post => <PostInfo post={post} key={post.id} />)}
```